src/index.html

```diff
@@ -7,8 +7,59 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="true"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="logo"
+      >
+        <img
+          src="/images/logo.png"
+          alt="Moyo logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#Laptops & Computers"
+              class="nav__link"
+              data-qa="hover"
+            >Laptops & Computers</a>
```

Comment:
[checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) p.6

Reply:
also, that's not how we move elements to a new line.

Here's the structure:
```
<tagName
  attribute1 // (2 indents for new nesting level)
  attribute2 / (2 indents for new nesting level)
  attribute // (2 indents for new nesting level)
> (closing brace, at the same level as opening one)
  Text  // (2 indents for new nesting level)
</tagName>
```